### PR TITLE
'For the Love of the Game' Blog (CCI)

### PR DESCRIPTION
**Pull request by uoduckswtd1**
### Submission Checklist:

* [X] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [X] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [X] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [X] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by aFinchy** [comment]
**What do you need to do?**
If you have a PR that is failing DCO checks, you can follow the instructions in the “Checks” tab for DCO to create a remediation commit or with these instructions:
1.    Create a new commit (can be empty, but should be avoided) with the following message:
```
DCO Remediation Commit for <Your Github name> <Your Github email>
I, <Your Github name> <Your Github email>, hereby add my Signed-off-by to this commit: <Unsigned commit hash>
Signed-off-by: <Your Github name> <Your Github email>
```
2.    If you have multiple commits that are not signed-off, you can batch them into the same commit, with the same message and different hash on each line
3.    Ensure this commit is signed off, then push the commit to your branch. The DCO check should now mark those commits as signed off.

File: content/blog/posts/for-the-love-of-the-game.md
Comment (sptramer) [inline content/blog/posts/for-the-love-of-the-game.md:15]: Images do not require the `/static` path in the link, and won't render if it's included.

```suggestion
![CCI-indie-developers](/images/blog/CCI/CCI-blog-image.png)
```

**Review by sptramer — CHANGES_REQUESTED** [review]

**Review by sptramer — APPROVED** [review]
Approved provided that the DCO is fixed for merge.

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:7]: Remove duplicate line.
`blog_img: "/images/blog/announcement_thumbnail.jpg"`

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:95]: Docs are internal, Change 
`[documentation](https://www.o3de.org/docs/) `
to
`[documentation](/docs/) `

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:79]: Doc link is internal, Change to internal link, not external
Change:
`[networking](https://docs.o3de.org/docs/user-guide/networking/)`
to
`[networking](/docs/user-guide/networking/)`

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:39]: Docs are internal, Change 
`[documentation](https://www.o3de.org/docs/) `
to
`[documentation](/docs/) `

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:13]: Change the Open 3D Engine link to Internal, not external.

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:8]: I would suggest having a Blog image instead of defaulting to the bland default image. This image is very outdated and needs a refresh.

**Review by aFinchy — CHANGES_REQUESTED** [review]
Left a bunch of formatting comments. 

The image that is in the same folder as the blog must be removed. Keep all images in the Static folder only.

File: content/blog/posts/for-the-love-of-the-game.md
Comment (aFinchy) [inline content/blog/posts/for-the-love-of-the-game.md:4]: Change date to the 21st

**Review by FiniteStateGit — APPROVED** [review]

**Review by OBWANDO — APPROVED** [review]

**Review by aFinchy — APPROVED** [review]
Fixed all the changes myself, and built locally as i found the image was also a bit too large. After all my changes, I approve now.

**Pull request merged**
